src/hooks/usePipelineRuns.ts

```diff
@@ -247,6 +247,7 @@ export const usePipelineRunsForCommit = (
   applicationName: string,
   commit: string,
   limit?: number,
+  filterByComponents = true,
```

Comment:
I noticed the `usePipelineRunsForCommit` hook is being used in a few places, so I decided to go with this approach of adding a new param instead of removing the filtering altogether

but, let me know if we should handle it differently :)